src/test/java/it/pn/frontend/e2e/stepDefinitions/common/BackgroundTest.java

```diff
@@ -24,6 +27,18 @@ public class BackgroundTest {
     private final HelpdeskTest helpdeskTest = new HelpdeskTest();
     private final NotifichePGPagoPATest notifichePGPagoPATest = new NotifichePGPagoPATest();
 
+    private static Map<String, String> datiPersonaFisica;
+    static
+    {
+        datiPersonaFisica = new HashMap<>();
+        datiPersonaFisica.put("nome", "Lucrezia");
+        datiPersonaFisica.put("cognome", "Borgia");
+        datiPersonaFisica.put("codiceFiscale", "BRGLRZ80D58H501Q");
+        datiPersonaFisica.put("ente", "Comune di Verona");
+        datiPersonaFisica.put("ragioneSociale", "Lucrezia Borgia");
+    }
```

Comment:
just put it without the **brackets** and the **static**

Reply:
i tried more clean insert data, but this is the best method to insert data in an HashMap which is accepted by the function that accepts that parameter